tools/gameserver/src/server.rs

```
@@ -119,6 +118,9 @@ impl Server {
                 let response = JoinResponse { rx, client_key, events };
                 let _ = join_tx.send(response);
             },
+            Request::Comment {inner} => {
+                self.tx.send(Event::Forward { inner: room_event::Event::Comment(inner) });
```

Please explicitly ignore the error here.